src/foam/dao/EnabledAware.java

```diff
@@ -7,12 +7,15 @@
 package foam.dao;
 
 import foam.mlang.predicate.Predicate;
+
+import java.sql.PreparedStatement;
 import java.util.Date;
 
 public interface EnabledAware {
   public final static Predicate ENABLED = new Predicate() {
     public boolean f(foam.core.FObject obj) { return true; }
-
+    public String createStatement(String table) { return ""; }
```

Comment:
Shouldn't this be "enabled = 't'"?

Reply:
I don't know. This was copied over from Carlos' PR

Reply:
Yes, it should be ;-).

Reply:
Okay. Added that in